de.metas.business/src/main/java/de/metas/pricing/service/ProductPrices.java

```diff
@@ -124,6 +130,29 @@ public static void assertMainProductPriceIsNotDuplicate(final I_M_ProductPrice p
 		getFirstOrThrowExceptionIfMoreThanOne(allMainPrices);
 	}
 
+	public static void assertUomConversionExists(final I_M_ProductPrice productPrice)
+	{
+		final IProductDAO productDAO = Services.get(IProductDAO.class);
```

Comment:
why were the checks here unnecessary? remember, you never know who is going to call this `public static` method.
Maybe you can annotate the productPrice parameter with `@NonNull` (lombok!), but i thik you should check still for IsActive and IsValid

Reply:
As it's only called in one place right now, it gets called at the same time as the assert before it (the one that checks that the product price is not duplicate), and the checks are removed are already gone through there.
But sure, for further use cases I could add them to be there.